mozfldp/model.py

```diff
@@ -95,6 +95,8 @@ def minibatch_update(self, X, y):
             init_intercept.shape, dtype=np.float64, order="C"
         )
 
+        self.set_training_classes(np.array(labels))
```

Comment:
This should actually be called once, externally by the user of the model object, rather than in the update function. Sorry, I realize this wasn't clearly documented. The flow I had in mind was:

- create a `SGDModel`
- call `set_training_classes` on the `labels` from the full dataset
- use the model in a simulation

In fact, the point of the SGDClassifier requiring the labels to be passed at some point is in case the dataset used for the `partial_fit` doesn't contain all the possible classes, which could very well be the case in a minibatch, so we would want to call it externally either way.